tools/accuracy_checker/accuracy_checker/adapters/detection_person_vehicle.py

```diff
@@ -299,3 +299,123 @@ def compute_iou_prop(lhs, rhs):
     @staticmethod
     def convert_classid_to_gt_type(class_id):
         return 0 if class_id == 5 else 1
+
+class PersonVehicleDetectionRefinementAdapter(Adapter):
+    __provider__ = 'person_vehilce_detection_refinement'
+    predcition_types = (DetectionPrediction, )
+
+    def process(self, raw, identifiers=None, frame_meta=None):
+        cls_names = ["bg", "bicycle", "bus", "car", "motorbike", "person", "van", "truck"]
+        thresholds = {1: 0.5, 2: 0.4, 3: 0.4, 4: 0.3, 5: 0.6, 6: 0.5, 7: 0.3}
+        kBoxNormalize = [10.0, 10.0, 5.0, 5.0]
+
+        detections = {'labels': [], 'scores': [], 'x_mins': [], 'y_mins': [], 'x_maxs': [], 'y_maxs': []}
+        proposals = frame_meta[0]['candidates']
+        img_height, img_width, _ = frame_meta[0]['image_size']
+        identifier = identifiers[0]
+
+        if proposals.x_mins.size == 0:
+            return [DetectionPrediction()]
```

Comment:
please do not forget to add identifier even for empty prediction and that adapter can process batch of inputs, not only single image